src/Xhgui/Controller/Run.php

```diff
@@ -108,6 +108,19 @@ public function view()
         ));
     }
 
+    public function delete()
+    {
+        $request = $this->app->request();
+        $id = $request->get('id');
+
+        $delete = $this->profiles->delete($id);
+
+        $this->_template = 'runs/delete.twig';
```

Comment:
Instead of rendering a template would it be more user friendly to redirect back to the list page, or the referring page?

Reply:
Yes redirecting back to referring page does make more sense :) I’ll look at changing that soon.